protoc-gen-frisbee/pkg/generator/client.go

```diff
@@ -0,0 +1,163 @@
+/*
+	Copyright 2022 Loophole Labs
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		   http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package generator
+
+import (
+	"github.com/loopholelabs/frisbee/internal/utils"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+const (
+	clientStruct  = "Client"
+	frisbeeClient = "*frisbee.Client"
+)
+
+func writeClientHandlers(f File, services protoreflect.ServiceDescriptors) {
+	operationIndex := 0
+	for i := 0; i < services.Len(); i++ {
+		service := services.Get(i)
+		for j := 0; j < service.Methods().Len(); j++ {
+			method := service.Methods().Get(j)
+			f.P(tab, "table[", operationIndex+operationOffset, "] = ", handlerSignature, " {")
```

Comment:
dear lord we should probably generate these with template files. I swear I wrote code to do this